src/meta-assistant/slack/slack-utils.ts

```diff
@@ -196,37 +191,32 @@ Have a great week!
    */
   export const postDailyMessageToUsers = async (
     dailyCombinedData: DailyCombinedData[],
-    timeRegistrations: TimeRegistrations[],
     previousWorkDays: PreviousWorkdayDates,
-    nonProjectTimes: NonProjectTime[]
   ): Promise<DailyMessageResult[]> => {
-    const { numberOfToday, yesterday, today } = previousWorkDays;
+    const { numberOfToday } = previousWorkDays;
 
     const messageResults: DailyMessageResult[] = [];
     for (const userData of dailyCombinedData) {
-      const { slackId, personId, expected } = userData;
-
-      const isAway = TimeUtilities.checkIfUserShouldRecieveMessage(timeRegistrations, personId, expected, today.toISODate(), nonProjectTimes);
-      const firstDayBack= TimeUtilities.checkIfUserShouldRecieveMessage(timeRegistrations, personId, expected, yesterday.toISODate(), nonProjectTimes);
-
+      const { slackId } = userData;
       const message = constructDailyMessage(userData, numberOfToday);
-
-      if (!isAway && !firstDayBack) {
-        if (!slackOverride) {
-          messageResults.push({
-            message: message,
-            response: await sendMessage(slackId, message.message)
-          });
-        }
-        else {
-          for (const stagingid of slackOverride) {
-            messageResults.push({
-              message: message,
-              response: await sendMessage(stagingid, message.message)
-            });
-          }
-        }
-      }
+      console.log("Message to send: ", message.message);
```

Comment:
For the PR this comment can be removed and the commented out code below should be uncommented

Reply:
And logs removed